### DISCUSSION TITLE
Chave inválida

### DISCUSSION THREAD

**Discussion by luizpaulosg7** [community/community#35322] [New to GitHub]
Olá,

Estou iniciando o curso CS50 e não estou conseguindo progredir porque a minha chave SSH é inválida.
Gostaria de saber como faço para conseguir uma chave válida para progredir com o curso.

Para ser mais especifico estou na aula: introdução > nossos canais > apresentação ID CS50 > Como testar seu código

Estou seguindo o vídeo e na etapa de ( Clique em Add SSH Key (Adicionar Chave SSH). está apresentando um erro, como esse (A chave é inválida. Você deve fornecer uma chave no formato de chave pública OpenSSH)

Pode me ajudar a resolver?

Desde já agradeço a atenção

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 

If the message you're receiving is saying that your SSH configuration is invalid, you should check the configuration file. Usually, it'll be in the file  `***/.ssh/config`, with *** representing your home directory.

**Comment by LucasHT22** [comment]
Hey @luizpaulosg7 , quando fiz o CS50 tive a mesma dúvida, porém descobri que precisa somente excluir a chave atual e gerar uma nova

**Comment by LucasHT22** [comment]
Confira: docs.github.com/pt/authentication/keeping-your-account-and-data-secure/reviewing-your-ssh-keys

**Comment by LucasHT22** [comment]
Isso deve ser útil, explica exatamente como criar https://docs.github.com/pt/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
